src/core/DataFile.cpp

```diff
@@ -794,6 +797,47 @@ void DataFile::upgrade_0_4_0_rc2()
 }
 
 
+void DataFile::upgrade_1_0_99()
+{
+	jo_id_t last_assigned_id = 0;
+	
```

Comment:
Empty lines should not have any indentation, but coding conventions are not against this.